chain/epoch_manager/README.md

```diff
@@ -0,0 +1,40 @@
+# near-epoch-manager crate
+
+Epoch manager crate is responsible for code related to epochs and epoch switching.
+An epoch is a unit of time when validators of the network remain constant.
+
+You can read more about the epoch here: https://docs.near.org/docs/concepts/epoch
+
+You can read more about Epoch finalization and Epoch changes here: https://github.com/near/NEPs/blob/master/specs/BlockchainLayer/EpochManager/EpochManager.md 
+
+## EpochManager
+Main class that has two main functions:
+* it creates new epochs (EpochIds)
+* allows accessing information about past and current epochs.
+
+### New Epoch Creation
+When 'finalize_epoch' is called, the EpochManager will do all the necessary processing (like computing validator rewards, selecting validators for the next epoch etc) and create the new EpochId/EpochInfo.
```

Comment:
We are actually selecting validators for epoch after the next at this point

Reply:
good catch - fixed.